app/src/main/java/framgia/com/ichat/data/source/PrivateRoomDataSource.java

```diff
@@ -1,15 +1,18 @@
-package framgia.com.ichat.data.source;
-
-import com.google.android.gms.tasks.OnCompleteListener;
-import com.google.android.gms.tasks.OnFailureListener;
-import com.google.firebase.database.ValueEventListener;
-
-public interface PrivateRoomDataSource {
-    interface Remote {
-        void getPrivateRooms(ValueEventListener valueEventListener);
-
-        void createPrivateRoom(OnCompleteListener onCompleteListener, OnFailureListener onFailureListener);
-
-        void deletePrivateRoom(String id, OnCompleteListener onCompleteListener, OnFailureListener onFailureListener);
-    }
-}
+package framgia.com.ichat.data.source;
+
+import com.google.android.gms.tasks.OnCompleteListener;
+import com.google.android.gms.tasks.OnFailureListener;
+import com.google.firebase.database.ValueEventListener;
+
+public interface PrivateRoomDataSource {
+    interface Remote {
+        void getPrivateRooms(ValueEventListener valueEventListener);
+
+        void createPrivateRoom(OnCompleteListener onCompleteListener,

```

Comment:
```suggestion
        void addRoom(OnCompleteListener onCompleteListener,
```
Tạo room mới mà k có gì để phân biệt sao 
ví dụ như name, id,.. ..

Reply:
cai nay e su dung key do firebase sinh ra

Reply:
vậy room mới ko có tên sao?

Reply:
cai nay e khoi tao 1 cai ten defalult va cho phep nguoi dung sua